src/utils/userLocation.ts

```diff
@@ -1,81 +1,129 @@
-import firebase, { UserInfo } from "firebase/app";
-
-import { Room } from "types/rooms";
+import firebase from "firebase/app";
 
 import { updateUserProfile } from "pages/Account/helpers";
 import { useInterval } from "hooks/useInterval";
 
+import { LOCATION_INCREMENT_MS, LOCATION_INCREMENT_SECONDS } from "settings";
+
 import { getCurrentTimeInMilliseconds } from "./time";
-import { openRoomUrl, openUrl, venueInsideUrl } from "./url";
+import { openRoomUrl } from "./url";
 
-const LOCATION_INCREMENT_SECONDS = 10;
-const LOCATION_INCREMENT_MS = LOCATION_INCREMENT_SECONDS * 1000;
+export type LocationData = Record<string, number>;
 
-type LocationData = Record<string, number>;
+export type UpdateLocationDataProps = {
```

Comment:
As per https://github.com/sparkletown/sparkle/pull/1062#issuecomment-772978074 this should use `interface` rather than `type`

Reply:
https://github.com/sparkletown/sparkle/pull/1129#discussion_r570165378

Reply:
As per https://github.com/sparkletown/sparkle/pull/1129#discussion_r570166887 I believe this is still a valid request, and aligns with the other changes made in this PR to align 'props' objects to using `interface`.